src/chigame/games/views.py

```diff
@@ -136,7 +136,8 @@ def test_func(self):
 
 
 def search_results(request):
-    query = request.GET.get("query")
+    # Must be "q" and not "query" as django machina framework expects a "q" input
```

Comment:
While certainly valuable to justify these changes now, I'm a bit iffy on the value of these comments for future developers. Once you are no longer looking at a diff with the old line, the comment loses some of its meaning and might even be confusing. Perhaps these could be reworded or maybe it would be better to include a comment with the element in base.html instead, where the context will be a little bit more clear.